relay-cabi/src/processing.rs

```diff
@@ -107,8 +107,11 @@ pub unsafe extern "C" fn relay_store_normalizer_normalize_event(
 ) -> RelayStr {
     let processor = normalizer as *mut StoreProcessor;
     let mut event = Annotated::<Event>::from_json((*event).as_str())?;
-    let config = LightNormalizationConfig::default();
-    light_normalize_event(&mut event, &config)?;
+    let light_normalization_config = LightNormalizationConfig {
+        normalize_user_agent: (*processor).config().normalize_user_agent,
```

Comment:
I don't remember, is `normalize_user_agent` the only config value in `StoreConfig` that applies to light normalization?

Reply:
Yes, it's used to determine to whether normalize it or not, see:

https://github.com/getsentry/relay/blob/ff0dd7290a827161f8bab35b33b1e9492c2d6a3d/relay-general/src/store/normalize.rs#L239

The user-agent normalization itself doesn't expect any parameters, it only takes the event, see:

https://github.com/getsentry/relay/blob/ff0dd7290a827161f8bab35b33b1e9492c2d6a3d/relay-general/src/store/normalize/user_agent.rs#L13